__tests__/app.test.js

```diff
@@ -194,3 +194,306 @@ describe("GET /api/articles/:article_id", () => {
+            comment_count: "2",
+            created_at: "2020-08-03T13:14:00.000Z",
+            title: "UNCOVERED: catspiracy to bring down democracy",
+            topic: "cats",
+            votes: 0,
+          },
+
+          {
+            article_id: 1,
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            comment_count: "11",
+            created_at: "2020-07-09T20:11:00.000Z",
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            votes: 100,
+          },
+
+          {
+            article_id: 9,
+            author: "butter_bridge",
+            body: "Well? Think about it.",
+            comment_count: "2",
+            created_at: "2020-06-06T09:10:00.000Z",
+            title: "They're not exactly dogs, are they?",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 10,
+            author: "rogersop",
+            body: "Who are we kidding, there is only one, and it's Mitch!",
+            comment_count: "0",
+            created_at: "2020-05-14T04:15:00.000Z",
+            title: "Seven inspirational thought leaders from Manchester UK",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 4,
+            author: "rogersop",
+            body: "We all love Mitch and his wonderful, unique typing style. However, the volume of his typing has ALLEGEDLY burst another students eardrums, and they are now suing for damages",
+            comment_count: "0",
+            created_at: "2020-05-06T01:14:00.000Z",
+            title: "Student SUES Mitch!",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 8,
+            author: "icellusedkars",
+            body: "Archaeologists have uncovered a gigantic statue from the dawn of humanity, and it has an uncanny resemblance to Mitch. Surely I am not the only person who can see this?!",
+            comment_count: "0",
+            created_at: "2020-04-17T01:08:00.000Z",
+            title: "Does Mitch predate civilisation?",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 11,
+            author: "icellusedkars",
+            body: "Having run out of ideas for articles, I am staring at the wall blankly, like a cat. Does this make me a cat?",
+            comment_count: "0",
+            created_at: "2020-01-15T22:21:00.000Z",
+            title: "Am I a cat?",
+            topic: "mitch",
+            votes: 0,
+          },
+
+          {
+            article_id: 7,
+            author: "icellusedkars",
+            body: "I was hungry.",
+            comment_count: "0",
+            created_at: "2020-01-07T14:08:00.000Z",
+            title: "Z",
+            topic: "mitch",
+            votes: 0,
+          },
+        ];
+        expect(articles).toEqual(articlesDesc);
```

Comment:
Just as a heads-up, jest-sorted is great for tests where you want to figure out whether something is ordered correctly